modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+
+[id="installation-generate-aws-user-infra-ignition_{context}"]
+= Creating the Kubernetes manifest and Ignition config files
+
+Because you must manually start the cluster machines, you must generate the
+Kubernetes manifest and Ignition config files that the cluster needs to make its
+machines.
+
+[IMPORTANT]
+====
+The Ignition config files that the installation program generates contain
+certificates that expire after 24 hours. You must complete your cluster
+installation and keep the cluster running for 24 hours in a non-degraded state
+to ensure that the first certificate rotation has finished.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} installation program.
+ifdef::restricted[]
+For a restricted network installation, these files are on your bastion host.
+endif::restricted[]
+* Create the `install-config.yaml` installation configuration file.
+
+.Procedure
+
+. Remove the Kubernetes manifest files for the control plane machines. By
+removing these files, you prevent the cluster from automatically generating
+control plane machines.
+.. Generate the Kubernetes manifests for the cluster:
```

Comment:
I'm a bit confused by this paragraph. First it talks about removing manifests, then it states to generate the manifests.

Reply:
> First it talks about removing manifests, then it states to generate the manifests.

You: "Installer, make your usual manifests." Installer: "Here you go" You: "Forget about these.  The rest look good.  Carry on."

However we want to explain that workflow works for me.

Reply:
This was pulled from the original module that I broke up, but I agree that it is confusing.

@kalexand-rh - I don't have the background to know why the steps are the way that they are. Do you have any reword suggestions that would make this clearer, or know who might be good to ask on this?

Reply:
Sorry I missed your comment @wking before I added mine - thanks for the explanation! 

@kalexand-rh let's look at this together tomorrow.